### PR TITLE
MNT: minimize lrose installation

### PR DESCRIPTION
**Pull request by kmuehlbauer**
I've tinkered a bit with installing lrose from scratch, with no success.

The bootstrapping process did'nt work (CMAKE and autotools based). It didn't find Netcdf and Qt headers/includes and the whole build process is hidden from the user. Not sure how to build without taking apart the complete python build scripts.

Nevertheless this PR cleans up a bit the installation of the lrose debian package. Not yurew how to test if it works correctly.

### CODE REVIEW

File: binder/appendix.txt [hunk 1]
```diff
@@ -8,12 +15,8 @@ RUN CONDA_DIR=/srv/conda && \
     echo "export RADARENV=$RADARENV" >> ~/.profile && \
     echo "export CONDA_PREFIX=$CONDA_PREFIX" >> ~/.profile && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
-    apt update && \
-    apt-get -y install sudo && \
-    sudo apt-get -y update && \
-    sudo -s && \
-    sudo ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/lrose/build_lrose.sh && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/nbstripout_notebooks.sh && \
```
Comment (mgrover1) [inline binder/appendix.txt:18]: Can we fix this typo here too?

**Review by mgrover1 — COMMENTED** [review]
One small suggestion here too

File: binder/appendix.txt (hunk 1 above)
Comment (kmuehlbauer) [reply binder/appendix.txt:18]: Oh, has it been there from the beginning?

Comment (mgrover1) [reply binder/appendix.txt:18]: I think so...

File: binder/appendix.txt [hunk 2]
```diff
@@ -8,12 +15,8 @@ RUN CONDA_DIR=/srv/conda && \
     echo "export RADARENV=$RADARENV" >> ~/.profile && \
     echo "export CONDA_PREFIX=$CONDA_PREFIX" >> ~/.profile && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
```
Comment (mgrover1) [inline binder/appendix.txt:17]:
```suggestion
    ${CONDA_PREFIX}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
```

**Review by mgrover1 — APPROVED** [review]

**Comment by kmuehlbauer** [comment]
@mgrover1 Not sure, if this was the only location of that typo. If you see fit, merge at your discretion. I'll have another look tomorrow. Thanks!

File: binder/appendix.txt [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
     echo "export RADARENV=$RADARENV" >> ~/.profile && \
     echo "export CONDA_PREFIX=$CONDA_PREFIX" >> ~/.profile && \
-    ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
-    apt update && \
-    apt-get -y install sudo && \
-    sudo apt-get -y update && \
-    sudo -s && \
-    sudo ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/lrose/build_lrose.sh && \
+    ${CONDA_PREFIX}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/nbstripout_notebooks.sh && \
```
Comment (mgrover1) [inline binder/appendix.txt:18]:
```suggestion
    ${CONDA_PREFIX}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/nbstripout_notebooks.sh && \
```

Comment (mgrover1) [reply binder/appendix.txt:18]: One more - sorry!

Comment (kmuehlbauer) [reply binder/appendix.txt:18]: Let me fix this tomorrow, with fresh eyes.

**Pull request merged**